doc/source/whatsnew/v1.5.3.rst

```diff
@@ -25,20 +25,21 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 - Bug in :meth:`.Styler.to_excel` leading to error when unrecognized ``border-style`` (e.g. ``"hair"``) provided to Excel writers (:issue:`48649`)
--
+- Bug in :meth:`.Timestamp.floor` fixed tz_localize so 'ambiguous' parameter can only be True, False, 'NaT' or 'raise' and all other invalid arguments to ambiguous have same error, including 'infer' (:issue:`49565`)
 
 .. ---------------------------------------------------------------------------
 .. _whatsnew_153.other:
 
 Other
 ~~~~~
--
+- Modified test cases in test_timezones.py to reflect new error message from bug fix for bug: (:issue:`49565`)
 -
 
 .. ---------------------------------------------------------------------------
 .. _whatsnew_153.contributors:
 
 Contributors
 ~~~~~~~~~~~~
+- Julia Chaker Aoun
```

Comment:
no need, this is populated automatically

Reply:
Hi! If you could approve or merge my PR by tomorrow I would really appreciate it! I'm working on this for a project and I get extra credit if the PR is approved :)

Reply:
will do if you address the review comments

Reply:
you'll need to address the review comments first

Reply:
still need to address https://github.com/pandas-dev/pandas/pull/50024?notification_referrer_id=NT_kwDOAf8KsLM0OTg0NTcwNTcxOjMzNDkxNjMy&notifications_query=reason%3Aparticipating#discussion_r1041506077 and https://github.com/pandas-dev/pandas/pull/50024?notification_referrer_id=NT_kwDOAf8KsLM0OTg0NTcwNTcxOjMzNDkxNjMy&notifications_query=reason%3Aparticipating#discussion_r1041505842